todo_app/data/trello_items.py

```diff
@@ -0,0 +1,247 @@
+
+    Output
+        id: id of the update item
+    """
+    target_url = f"https://api.trello.com/1/cards/{card_id}"
+
+    params = {
+        "key": apikey,
+        "token": token,
+        "idList": destination
+    }
+
+    api_response = call_api(target_url, "PUT", params)
+
+    return id
+
+
+def remove_item(card_id, apikey, token):
+    """
+    Removes an existing item from the session. If no item matching the id then nothing is returned
+
+    Args:
+        card_id: item id to be removed
+        apikey: apikey for accessing trello
+        token: api token for accessing trello
+
+    Returns:
+        id: item id to be removed
+
+    """
+    target_url = f"https://api.trello.com/1/cards/{card_id}"
+
+    params = {
+        "key": apikey,
+        "token": token
+    }
+
+    api_response = call_api(target_url, "DELETE", params)
+
+    return card_id
+
+
+def get_list_names(board_id, apikey, token):
+    """Get a list of all list names within the trello board
+
+    Args
+        board_id: id of the board we are going to use
+        apikey: trello api key
+        token: trello token
+
+    output
+        target_lists: a list of all list ids from the trello board we are interested in
+    """
+
+    target_url = f"https://api.trello.com/1/boards/{board_id}/lists"
+
+    params = {
+        "key": apikey,
+        "token": token
+    }
+
+    api_response = call_api(target_url, "GET", params)
+
+    target_lists = {"To Do": None,
+                    "Done": None}
+
+    #loop over the lists to get the list names that we need
+    for list in api_response:
+        if list.get("name") in target_lists.keys():
+            target_lists[list.get("name")] = list.get("id")
+
+    for k, v in target_lists.items():
+        if not v:
+            raise Exception(f"No list named {k}. Please create a list with this name")
+
+    return target_lists
+
+
+if __name__ == "__main__":
+
+    load_dotenv()
+
+    board_id = os.getenv("BOARD_ID")
+    apikey = os.getenv("APIKEY")
+    token = os.getenv("TOKEN")
+
+    board_lists = get_list_names(board_id, apikey, token)
+
+    get_items_response = get_items(board_id,
+                                   apikey,
+                                   token)
+
+    print(get_items_response)
+
+    response = get_items("62cc1dd1e32d46579e3e2218",
+              apikey,
+              token)
+    print(response)
+
+    add_response = add_item("Create a new test item",
+                            "This should create a new test item",
+                            "2022-08-23",
+                            apikey,
+                            token)
+
+    change_item_status(get_items_response[0].get("id"),
+                       board_lists.get("done"),
+                apikey,
+                token)
+
+    delete_response = remove_item(get_items_response[0].get("id"),
+                                  apikey,
+                                  token)
+
+
```

Comment:
There is lots of unnecessary white space here. On our project, we will have a script to automatically format the python files to avoid this.